lms/djangoapps/certificates/api.py

```diff
@@ -63,3 +72,108 @@ def certificate_downloadable_status(student, course_key):
         response_data['download_url'] = current_status['download_url']
 
     return response_data
+
+
+def cert_generation_enabled(course_key, is_enabled=None):
```

Comment:
I don't know if this is a Python convention or what, but `cert_generation_enabled` sounds more like it only returns the current `is_enabled` status. Maybe something more like `set_cert_generation_enabled`?